netlify.toml

```diff
@@ -1,3 +1,7 @@
 [build]
   publish = "_site"
   command = "gem install bundler && .ci/build.sh"
+
+[[redirects]]
+  from = "/#/projects?project=integrate_pyflakes-enhanced_ast_into_coala"
```

Comment:
to & from need to be reversed.

and you need to edit the .md to remove the unwanted space

Reply:
The link with _ is the working one, so I redirected the one without _ (not working), to the one with _